### DISCUSSION TITLE
[Enhancement] Create an official .NET MAUI blog

### DISCUSSION THREAD

**Discussion by giuseppenovielli** [dotnet/maui#111] [General] [answered]
## Summary
Provide an official blog like Xamarin Blog https://devblogs.microsoft.com/xamarin/
that contains small article and all contents, link, references about all .Net MAUI update and progress.

## Intended Use Case
Developer's can be stay up to date, about all .NET MAUI.

**Comment by davidortinau** [comment, marked as answer]
We'll definitely be blogging! Initially I expect it'll be a combination of the .NET and Xamarin blogs where we have an existing audience.
